.template/addons/github/.github/workflows/test.yml.tt

```diff
@@ -0,0 +1,76 @@
+name: Test
+
+on: push
+
+env:
+  DOCKER_REGISTRY_HOST: ${{ secrets.DOCKER_REGISTRY_HOST }}
+  DOCKER_REGISTRY_USERNAME: ${{ github.repository_owner }}
+  DOCKER_REGISTRY_TOKEN: ${{ secrets.DOCKER_REGISTRY_TOKEN }}
+  COMPOSE_FILE: docker-compose.test.yml
+  DOCKER_IMAGE: ${{ github.repository }}
+
+jobs:
+  build:
+    name: Build docker image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Cancel previous runs
+        uses: styfle/cancel-workflow-action@0.5.0
+        with:
+          access_token: ${{ github.token }}
+
+      - uses: actions/checkout@v2.3.2
+
+      - uses: nimblehq/branch-tag-action@v1.1
+
+      - name: Docker login
+        run: echo "$DOCKER_REGISTRY_TOKEN" | docker login $DOCKER_REGISTRY_HOST --username=$DOCKER_REGISTRY_USERNAME --password-stdin
+
+      - name: Docker pull
+        if: ${{ env.BRANCH_TAG != 'latest' && env.BRANCH_TAG != 'development' }}
+        run: docker-compose pull test || true
+
+      - name: Docker build
+        run: |
+          bin/docker-prepare
+          docker-compose build
+
+      - name: Docker push
+        run: |
+          docker-compose push test
```

Comment:
why don't we just create new steps here to run the tests instead of creating new jobs to do that?
```yml
- name: Run Unit Tests
   run: docker-compose run test bundle exec rspec --exclude-pattern "spec/systems/**/*_spec.rb" --profile
- name: Run System Tests
   run: docker-compose run test bundle exec rspec spec/systems --profile
```